src/app/services/suggestion.service.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { TestBed } from '@angular/core/testing';
+
+import { SuggestionService } from './suggestion.service';
+import { NotificationService } from './notification.service'
+import { ConnectionService } from './connection.service';
+import { LocalNotifications } from '@ionic-native/local-notifications/ngx';
+import { UniqueDeviceID } from '@ionic-native/unique-device-id/ngx';
+
+describe('SuggestionService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [NotificationService, LocalNotifications, UniqueDeviceID, ConnectionService]
+    })
+
+  });
+
+  it('should be created', () => {
+    const service: SuggestionService = TestBed.get(SuggestionService);
+    expect(service).toBeTruthy();
+  });
+
+  // TODO: Add tests
```

Comment:
Are you waiting on the implementation of server functions to write these tests? If not, you **must** write these tests.

Reply:
I'll implement the test later. Please merge so that @TheSnekySnek can implement moderator functions.